### ISSUE TITLE
Support for weekly indices

### ISSUE THREAD

**Issue by profilernz** [etsy/411#66]
For logstash, we use:
index => "stats-%{+xxxx.ww}"
and
config.php mentions:
'date_based' => true, # Whether to append a date to the index (`logstash` becomes `logstash-2016.01.01`).
So it seems there is no current support for those logstashing to weekly indices?

**Comment by kiwiz** [comment]
Not at the moment. I'd like to add support for arbitrary expressions, but that's a big chunk of work and low priority (It would require implementing all of this: http://joda-time.sourceforge.net/apidocs/org/joda/time/format/DateTimeFormat.html)

A workaround is to specify:
```
    'date_based' => false, # Tells 411 not to modify the index name.
    'index' => 'stats-*',
```

**Comment by profilernz** [comment]
That workaround is fine, thanks.

**Comment by kiwiz** [comment]
Implemented in this (poorly named) commit 61894305e6c55a3517a809f6d81613890027875b.
